### ISSUE TITLE
Embedded workflow not recognized by Civitai?

### ISSUE THREAD

**Issue by Tinase-nau** [alexopus/ComfyUI-Image-Saver#53]
Is the embedded workflow meant for something else except for displaying by "Workflow: Nodes" at Civitai? I noticed it doesn't show up there at all. Never has for me. It would be cool if it was possible to share as it helps newbies with workflows.

**Comment by alexopus** [comment]
The full workflow embedding is a core ComfyUI feature, and what civitai does with it is in the hands of civitai. This node set does not extend those features.

**Comment by Tinase-nau** [comment]
embed_workflow_in_png on Image Saver node - is it supposed to work the same way as Comfy Core's Save Image? When I do save the image with CC's node, then the workflow is embedded and Civitai displays the metadata. When I use ComfyUI-Image-Saver's node then Civitai doesn't display anything. So that's my actual question: does it really work?

**Comment by alexopus** [comment]
- Are you saving it as png?
- Can you load the workflow from the created file in Comfy?

**Comment by Tinase-nau** [comment]
Yes, out is PNG and the saved workflow can be successfully imported into ComfyUI.
Here are the output PNG and the workflow:
![125764130183399, pixelAlchemy_pixelAlchemy30](https://github.com/user-attachments/assets/1fec8c3a-67af-4f71-9733-27393d7404c9)
[125764130183399, pixelAlchemy_pixelAlchemy30.json](https://github.com/user-attachments/files/18292829/125764130183399.pixelAlchemy_pixelAlchemy30.json)

**Comment by alexopus** [comment]
If it loads fine into Comfy then everything is correct.
Most likely civitai does not recognize something in the workflow and refuses to store it as is.
Nothing to be done here i'm afraid.

**Comment by Tinase-nau** [comment]
If to take the default ComfyUI workflow and just replace the core image saver with yours, then workflow is not displayed by Civitai.
Something is broken specifically with the node itself. Does it work for you though? Or is my setup corrupted...

**Comment by alexopus** [comment]
It does not work for me and it never did. Civitai only recognizes very simple flows.
My guess is that it will refuse to accept it if it encounters an unknown node, which could be the image saver node.
It's not something that is wrong with this node, it is a limitation on civitai side, possibly intentional to not accept nodes they don't know.

**Comment by Tinase-nau** [comment]
I dug around to see if... 
I found an image by a chinese fellow with its workflow presented by Civitai. It had custom nodes from several nodes packages. It used the WAS Node Suite's Image Save node to embed data.
https://civitai.com/images/48751551
The node is able to save the workflow. But it has its issues - instead of using direct wiring, it uses fetching "over the air" and if it doesn't find what it is looking for it can't include that metadata, for e.g.: model and lora info. And when I tried to include WAS' node in my workflow, it couldn't embed any metadata at all...
The chinese fellow's complex workflow worked for it and something works with the core nodes, that's a start to take a look into. 
But then again you may be right and Civitai may have limitations.
